content/accessibility/accessibility-at-github.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Accessibility at GitHub
```

Comment:
I wonder if, to simplify the IA, this content should be the a11y landing page, what do you think @ichelsea @matthewferry

Reply:
@yaili Can you clarify? This is the "landing" page for the accessibility content. For lack of side-bar structure, it's just grouped into the four pages.

Reply:
I think I meant that when you click on the top level "Accessibility" title in the navigation this is where this content should be, instead of having a separate page underneath. Right now the other top level sections (Global, UI patterns, etc.) are empty, but that needs to be corrected.

Reply:
Oh wow I did not even realize those were selectable. I like that idea, but I want to make sure that these are obviously links to click so that people don't miss this important info. If we put GitHub's stance on accessibility in design hidden behind something that's not super accessible that won't look great 😅

Reply:
that's a good point, maybe at some point we need to add an "overview" page that goes to the same place as the top level links